src/App.css

```diff
@@ -27,6 +27,28 @@
   display: inline;
 }
 
+.App .spaceyspace {
+  height: 60px;
+}
+
+.App .exiting {
+  animation: comp-exit 2s linear;
+}
+
+.App .entering {
```

Comment:
Using animation frames is fine here but you could probably just use transition as well, but everything is well formed except the classNames. Make sure to enter something useful in className for the <Transition> component. If you put in "questions" for className you'd get .questions-entering .questions-entered .questions-exiting and .questions-exited as your classes